api-tester/src/main/java/com/revenuecat/apitester/kotlin/PurchasesAPI.kt

```diff
@@ -86,16 +85,14 @@ private class PurchasesAPI {
         purchases.purchasePackage(activity, packageToPurchase, upgradeInfo, purchaseChangeCallback)
         purchases.purchasePackage(activity, packageToPurchase, purchaseCallback)
 
-        purchases.purchaseProductOption(activity, storeProduct, purchaseOption, upgradeInfo, purchaseChangeCallback)
-        purchases.purchaseProductOption(activity, storeProduct, purchaseOption, purchaseCallback)
-        purchases.purchasePackageOption(
+        purchases.purchaseSubscriptionOption(
             activity,
-            packageToPurchase,
+            storeProduct,
             purchaseOption,
             upgradeInfo,
             purchaseChangeCallback
         )
-        purchases.purchasePackageOption(activity, packageToPurchase, purchaseOption, purchaseCallback)
+        purchases.purchaseSubscriptionOption(activity, storeProduct, purchaseOption, purchaseCallback)
```

Comment:
i wonder if we should rename `PurchaseOption` -> `SubscriptionOption`?

Reply:
Hmm I like it, since `PurchaseOption` will only be related to subscriptions this way 👍. Probably we can do that in a different PR yeah.